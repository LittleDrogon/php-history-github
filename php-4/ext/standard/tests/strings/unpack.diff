001- Warning: unpack(): Invalid format type - in %s/unpack.php on line %d
001+ Warning: unpack() [/phpmanual/function.unpack.html]: Invalid format type - in /home/star/Desktop/php-history/php-4/ext/standard/tests/strings/unpack.php on line 2