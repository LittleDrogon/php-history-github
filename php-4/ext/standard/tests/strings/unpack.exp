Warning: unpack(): Invalid format type - in %s/unpack.php on line %d
bool(false)
Done