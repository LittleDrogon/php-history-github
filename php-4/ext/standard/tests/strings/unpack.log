
---- EXPECTED OUTPUT
Warning: unpack(): Invalid format type - in %s/unpack.php on line %d
bool(false)
Done
---- ACTUAL OUTPUT
Warning: unpack() [/phpmanual/function.unpack.html]: Invalid format type - in /home/star/Desktop/php-history/php-4/ext/standard/tests/strings/unpack.php on line 2
bool(false)
Done
---- FAILED
