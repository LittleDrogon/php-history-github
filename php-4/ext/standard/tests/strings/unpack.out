Warning: unpack() [/phpmanual/function.unpack.html]: Invalid format type - in /home/star/Desktop/php-history/php-4/ext/standard/tests/strings/unpack.php on line 2
bool(false)
Done